composer.json

```
@@ -59,8 +59,25 @@
 		],
 		"check-complete-strict": [
 			"@php ./vendor/phpcsstandards/phpcsdevtools/bin/phpcs-check-feature-completeness ./WordPress"
+		],
+		"test": [
+			"@lint",
+			"@check-cs",
+			"@run-tests",
+			"@check-complete-strict"
 		]
 	},
+	"scripts-descriptions": {
+		"lint": "Lint PHP files against parse errors.",
+		"lint-ci": "Lint PHP files against parse errors, but output the results as Checkstyle XML (used for CI purposes).",
+		"check-cs": "Run the PHPCS script against the entire codebase.",
+		"fix-cs": "Run the PHPCBF script to fix all the autofixable violations on the codebase.",
+		"install-codestandards": "Manually install the coding standards, so that they are discoverable by PHPCS script. Used mostly for CI purposes",
+		"run-tests": "Run all the unit tests for the WordPress Coding Standards sniffs.",
+		"check-complete": "Check if all the sniffs are feature complete (has unit tests and XML documentation), \n but don't warn if the documentation is missing for the sniffs, or if there are documentation and test files present that are not associated with a particular sniff.",
```

What about simplifying it to describe just what it does ? (and not what it doesn't do)
```suggestion
		"check-complete": "Check if all the sniffs have tests.",
```

---

Yup, this is definitely better 😄 👍🏼